client/src/partials/accounts/accounts.js

```diff
@@ -24,6 +25,9 @@ function AccountsController($rootScope, AccountGrid, Notify, Constants) {
   // this flag will determine if the grid should expand the rows on data change
   vm.initialDataSet = true;
 
+  // lang parameter for document
+  vm.parameter = { lang: Language.key };
```

Comment:
I wonder if we should just modify the `bhPdfPrint` component to send back the language automatically.  That would clean up our code quite a bit.